server/test/uk/gov/ons/addressIndex/server/modules/ElasticsearchRepositorySpec2.scala

```diff
@@ -0,0 +1,53 @@
+package uk.gov.ons.addressIndex.server.modules
```

Comment:
I'm saying that I already implemented those tests and there is no need in additional ones in my opinion.